test/src/test/java/hudson/security/HudsonPrivateSecurityRealmTest.java

```diff
@@ -559,6 +561,24 @@ public void ensureHashingVersion_2y_isNotSupported() {
         assertThrows(IllegalArgumentException.class, () -> BCrypt.checkpw("a", "$2y$08$cfcvVd2aQ8CMvoMpP2EBfeodLEkkFJ9umNEfPD18.hUF62qqlC/V."));
     }
 
+    @Test
+    public void hashedPasswordTestPBKDF2() {
```

Comment:
It looks like partially someone tried to follow: [state under test][expected behaviour] unit tests naming convention here in this file. Please try to rename your tests accordingly. In particular word "Test" is not used in other tests names.